### ISSUE TITLE
I have PHP 8.1 and Laravel 10. My error is in the Model: Undefined type: "\OwenIt\Auditing\Auditable"

### ISSUE THREAD

**Issue by evolucionaurea** [owen-it/laravel-auditing#863]
| Q                 | A
| ----------------- | ---
| Bug?              | no|yes
| New Feature?      | no|yes
| Framework         | Laravel|Lumen
| Framework version | 8.x.y
| Package version   | 12.x.y
| PHP version       | 7.x.y

#### Actual Behaviour

Describe the behaviour you're experiencing. Do not just copy and paste a random error message and expect help.

#### Expected Behaviour

Describe the behaviour you're expecting.

#### Steps to Reproduce

List **all the steps** needed to reproduce the issue you're having.

Make sure to **include code** (affected models, configuration, ...).

Pastebins, screenshots and/or other resources that may help us understand what's going on, will also be appreciated.

In a nutshell, help us, help you.

#### Possible Solutions

If you have any ideas on how to solve the issue, add them here, otherwise you can omit this part.

**Comment by parallels999** [comment]
>Describe the behaviour you're experiencing. Do not just copy and paste a random error message and expect help.
>List all the steps needed to reproduce the issue you're having.
Make sure to include code (affected models, configuration, ...).
Pastebins, screenshots and/or other resources that may help us understand what's going on, will also be appreciated.
In a nutshell, help us, help you.

You give too much information to investigate.

**Comment by evolucionaurea** [comment]
**Versions:**
PHP: 8.1
owen-it/laravel-auditing: "^13.5"

<img width="352" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/0e0bed19-2458-4cba-9b30-07655141cc03">


**Vendor Folder:**
The package is installed correctly

<img width="286" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/b4af401a-7ef1-481d-8f24-b33917a481c2">


**Config**
In config/app.php

<img width="811" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/4b1f7b1c-f686-4270-8d11-548056063196">


**My error:**
VisualStudio tells me an error in a Model
<img width="1025" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/3d360c45-5bdd-4ac2-8c91-8c708ddda777">


**Tests I did**
I have a seeder to create a "Solicitud".
When I run the seeder it only records in the "Solicitud" table but not in Audit

<img width="913" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/3293ed5e-0e65-4168-a40a-2994a24bf890">


<img width="761" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/d3a62b11-c9d6-42ca-9fc4-ec5397748acd">

**Comment by parallels999** [comment]
>Config
>In config/app.php

It is not necessary to add that line in the config file, the package uses autodiscover

**Comment by evolucionaurea** [comment]
**1) Delete the line in providers**

<img width="449" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/1a01d5a6-3e46-4574-8cc2-a977b352469e">


**2) This is my Seeder:**
<img width="491" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/e55fcf3d-2dd6-4ad1-835a-de9f5d6d2700">

**3) I run composer dump-autoload ok**

**4) Does not save in the audits table**
<img width="830" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/2301a528-cdf5-4777-8e86-02d70c89d55e">


**5) The warning still appears in the model**
<img width="716" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/e4633c99-4b0f-4cea-950f-828d8c55bdd4">


**6) Could it be that Audit is not compatible with PHP 8.1?**
<img width="406" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/39563f0c-6465-41d1-a99c-00a86e4e1451">

**Comment by evolucionaurea** [comment]
In my Seeder try "Insert" but it doesn't work either

**Comment by parallels999** [comment]
> Could it be that Audit is not compatible with PHP 8.1?

I'm using it on 8.1, 8.2, and everything works, but i never try to audit seeders

>VisualStudio tells me an error in a Model

That looks like another problem.

**Comment by erikn69** [comment]
do you have `audit.console => true`?
https://github.com/owen-it/laravel-auditing/blob/7d929c01cad48be1785ae8c7160761d390e2f46b/src/Auditable.php#L525-L527

**Comment by evolucionaurea** [comment]
I was doing tests.
Apparently it doesn't work with seeders but using postman it works fine!

**Comment by evolucionaurea** [comment]
thank you so much

**Comment by evolucionaurea** [comment]
Hi.
I leave an image of how I managed to make it work.
In the seeder I tried "insert" and "create" but it didn't work.
Creating a new instance of the model and using the "save()" function worked correctly

<img width="388" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/81f40a11-cad8-4f15-9fdf-9b5ee6fc0937">

Bye

**Comment by parallels999** [comment]
@evolucionaurea did you try `$cliente->fill($request->all());`, if you have 100 properties and if you do it one at a time, it will take you a lot

**Comment by evolucionaurea** [comment]
Yeah.
I see that from the controller it always works fine.
Using insert() or create() works correctly.

The Seeder is where it doesn't work.

**In controller this is ok:**
$cliente = Cliente::findOrFail($id);
$cliente->update($data);

**Comment by parallels999** [comment]
> do you have audit.console => true?

That didn't work either?

**Comment by evolucionaurea** [comment]
Hi.
There I set console to true and now they work correctly with the seeders.
Thank you!

<img width="723" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/18aee42f-4e3b-4521-8e67-a603551ad9fc">


<img width="738" alt="image" src="https://github.com/owen-it/laravel-auditing/assets/25930048/3e8b0914-6eb5-4d80-ab8c-6b854406bf91">

**Comment by evolucionaurea** [comment]
Sorry, I was dealing with a lot of things at the same time. I must not have seen her.
Thank you so much!
